### ISSUE TITLE
Add aria tags to generated svgs 

### ISSUE THREAD

**Issue by barlock** [IBM-Design/icons#48]
SVGS could use an `aria-labeled-by="title"` with a title for better accessibility on screen readers.

**Comment by seejamescode** [comment]
Yeah, I just read this article about [accessible SVGs](https://css-tricks.com/accessible-svgs/#article-header-id-6). It seems like our responsibility in this code is to add `<title></title>` inside of built SVGs.

It also sounds like we should recommend the use of `aria-label` and `aria-hidden` for their appropriate purposes in the Readme. Agreed?
